examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -39,6 +40,16 @@ RegularGridDecomposition::RegularGridDecomposition(const MDFlexConfig &configura
 
   _decomposition = DomainTools::generateDecomposition(_subdomainCount, configuration.subdivideDimension.value);
 
+  // For reflection, we interact particles with their 'mirror' only if it would cause a repulsive force. This occurs
+  // only for particles within sixthRootOfTwo * sigma / 2.0 of the boundary. For minimal redundant iterating over
+  // particles, we iterate once over particles within the largest possible range where a particle might experience a
+  // repulsion, i.e. sixthRootOfTwo * maxSigma / 2.0.
```

Comment:
Where do we/you know this from?

Reply:
Ask Sam, he wrote that :P
If you look at the LJ Potential, the distance where attraction flips to repulsion this happens at $\sqrt[6]{2} \sigma$. So for a particle to have its mirror particle in that distance, it must be $\sqrt[6]{2} \sigma / 2$ away from the mirror.